src/main/java/teammates/ui/controller/InstructorStudentListPageCourseData.java

```diff
@@ -0,0 +1,25 @@
+package teammates.ui.controller;
+
+import teammates.common.datatransfer.CourseAttributes;
+
+/**
+ * Serves as a datatransfer class between {@link InstructorStudentListPageAction}
+ * and {@link InstructorStudentListPageData}
+ */
+public class InstructorStudentListPageCourseData {
```

Comment:
In the end I made a new package ui.datatransfer. I know there is common.datatransfer but I don't think it fits very well there.